assets/sass/06-components/entry.scss

```diff
@@ -8,8 +8,11 @@
 
 	a {
 		color: var(--entry-header--color-link);
+		text-decoration: none;
 
 		&:hover {
+			text-decoration: underline dotted;
```

Comment:
This shorthand is being discarded in Safari (v14.0). Works fine in Chrome and FF, but I think we need to separate it out for now. 

![Screen Shot 2020-10-14 at 9 06 15 AM](https://user-images.githubusercontent.com/1202812/95993106-aec50800-0dfc-11eb-8576-3576bfd265f8.png)

Reply:
Huh... Didn't know about that, thanks!
Separated.
I'll create a different PR because I see we use the same shorthand in other places too.